src/main/java/com/googleinterns/zoomtube/servlets/TranscriptServlet.java

```diff
@@ -69,7 +69,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String videoId = request.getParameter(LectureUtil.VIDEO_ID);
     Document document = getTranscriptXmlAsDocument(videoId).get();
     long lectureId = Long.parseLong(request.getParameter(LectureUtil.ID));
-    putTranscriptLinesInDatastore(lectureId, document);
+    Key lectureKey = KeyFactory.createKey(LectureUtil.KIND, lectureId);
```

Comment:
nit: I'm wondering if  ```LectureUtil.KIND``` needs to be named ```LectureUtil.KEY```?

Reply:
I feel `KIND` would work better because `KIND` is also used to indicate the type of entity.